src/ray/raylet/node_manager.cc

```diff
@@ -587,7 +587,7 @@ void NodeManager::FillResourceReport(rpc::ResourcesData &resources_data) {
   resources_data.set_node_manager_address(initial_config_.node_manager_address);
   // Update local cache from gcs remote cache, this is needed when gcs restart.
   // We should always keep the cache view consistent.
-  cluster_resource_scheduler_->GetLocalResourceManager().UpdateLastResourceUsage(
+  cluster_resource_scheduler_->GetLocalResourceManager().ResetResources(
```

Comment:
I feel this name is confusing: for local resource, current node should be the source of truth. Why do we need to update local resource usage from gcs client?

Reply:
I think the better name should be ResetLastReportResourceUsage or something similar.